internal/pkg/deploy/cloudformation/app.go

```diff
@@ -214,29 +229,29 @@ func (cf CloudFormation) RemoveServiceFromApp(app *config.Application, svcName s
 	}
 
 	// We'll generate a new list of Accounts to remove the account associated
-	// with the input service to be removed.
-	var svcList []string
-	shouldRemoveSvc := false
-	for _, svc := range previouslyDeployedConfig.Services {
-		if svc == svcName {
-			shouldRemoveSvc = true
+	// with the input workload to be removed.
+	var wlList []string
+	shouldRemoveWl := false
+	for _, wl := range previouslyDeployedConfig.Services {
+		if wl == wlName {
+			shouldRemoveWl = true
 			continue
 		}
-		svcList = append(svcList, svc)
+		wlList = append(wlList, wl)
 	}
 
-	if !shouldRemoveSvc {
+	if !shouldRemoveWl {
 		return nil
 	}
 
 	newDeploymentConfig := stack.AppResourcesConfig{
 		Version:  previouslyDeployedConfig.Version + 1,
-		Services: svcList,
+		Services: wlList,
 		Accounts: previouslyDeployedConfig.Accounts,
 		App:      appConfig.Name,
 	}
 	if err := cf.deployAppConfig(appConfig, &newDeploymentConfig); err != nil {
-		return fmt.Errorf("removing %s service resources from application: %w", svcName, err)
+		return fmt.Errorf("removing %s service resources from application: %w", wlName, err)
```

Comment:
Can this be reworded to `job` or perhaps the following?
```suggestion
		return fmt.Errorf("removing resources for %s from application: %w", wlName, err)
```